test/integration/content-preview/Annotations.e2e.test.js

```diff
@@ -92,4 +92,18 @@ describe('ContentPreview with Annotations', () => {
         // Current version header
         cy.get('.bcpr-PreviewHeader').should('be.visible');
     });
+
+    it('Should show drawing annotations', () => {
+        helpers.load({
+            fileId: Cypress.env('FILE_ID_DOC_ANNOTATIONS'),
+            props: {
+                boxAnnotations: new global.BoxAnnotations(null, { features: { drawing: true } }),
+            },
+        });
+
+        cy.get('.ba-DrawingTarget')
```

Comment:
Is there a `data-testid` attribute we can use?

Reply:
The `data-testid` is like `ba-AnnotationTarget-xxx` which is not Drawing specific.